e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -248,6 +257,66 @@ export function createTextCard({
   };
 }
 
+export function getHeadingCardDetails({
+  id = getNextUnsavedDashboardCardId(),
+  col = 0,
+  row = 0,
+  size_x = 24,
+  size_y = 1,
+  text = "Heading text example dto",
```

Comment:
```suggestion
  text = "Heading text details",
```

Reply:
Realized I left this "dto" reference there, feel free to click on "commit suggestion" when reviewing